docs/using.rst

```diff
@@ -18,69 +18,24 @@ recommended that you do the following:
 
 * Back up your code.
 * Use a source control system.
-* Mount your code into the box from your host system. The provided `custom Vagrantfile examples
-  <#using-custom-vagrantfiles-for-specific-use-cases>`_ already mount your home directory on the host
-  into the guest using `vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs/>`_.
+* Mount your code into the box from your host system. The provided `custom Vagrantfile examples <docs/installing.rst#4-download-the-adb>`_
+  already mount your home directory on the host into the guest using `vagrant-sshfs <https://github.com/dustymabe/vagrant-sshfs/>`_.
   Additional shares can also be defined. Refer to the `bi-directional folder sync
   section <#vagrant-bi-directional-folder-sync>`_ for more information.
 
 Doing these things is beyond the scope of this document. Consult your Operating
 System manuals and the `Vagrant <http://vagrantup.com/>`_ website for more
 details.
 
-Starting the Vagrant Box
-========================
+Once you start the ADB you can use it as:
```

Comment:
+1 to removing "the" from ADB, @bexelbie this is what I was talking about in one of our initial conversations, in general product names do not incur articles (not just in CDK/RH), so I support standardizing on non-"the".